ui/menu-workItemToolbarButton.html

```diff
@@ -0,0 +1,78 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Work item menu contribution sample</title>
+</head>
+
+<body>
+    <script src="sdk/scripts/VSS.SDK.js"></script>
+    <script>
+
+        VSS.init({ usePlatformScripts: true });
+
+        // Get the ActiveWorkItemService.  This service allows you to get/set fields/links on the 'active' work item (the work item
+        // that currently is displayed in the UI).
+        function getActiveWorkItemService()
+        {
+            var deferred = $.Deferred();
+            
+            VSS.require(["TFS/WorkItemTracking/Services"], function(_WorkItemServices) {
+                _WorkItemServices.ActiveWorkItemService.getService().then(function (service) {
+                    deferred.resolve(service)
+                },
+                function() {
+                    deferred.reject();
+                });
+            });
+
+            return deferred.promise();           
+        }
+
+
+        // Register a listener for the menu item contribution
+        VSS.register("MenuItemListener", function (context) {
+            return {
+                setFields: function(properties, title) {
+                    
+                    // Get the active work item service
+                    getActiveWorkItemService().then(function (service) {
+                        // Set a field value
```

Comment:
It would be good to add more comments here. Indicate we are using refnames for fields, and maybe a link to the documentation or a link to the list of available refnames for system fields